_attributes/common-attributes.adoc

```diff
@@ -23,12 +23,12 @@
 :op-system-version-9: 9
 :op-system-ai: Red{nbsp}Hat Enterprise Linux AI
 ifdef::openshift-origin[]
-:op-system-first: Fedora CoreOS (FCOS)
-:op-system: FCOS
-:op-system-lowercase: fcos
-:op-system-base: Fedora
-:op-system-base-full: Fedora
-:op-system-version: 35
+:op-system-first: CentOS Streams CoreOS (SCOS) 
```

Comment:
It is "CentOS Stream" i believe, not "Streams". @ausil can correct me if i'm wrong.

Reply:
It [appears you are correct](https://www.centos.org/centos-stream/). Thanks!